app/src/main/java/com/onboarding/calculator/mvp/model/CalculatorModel.java

```diff
@@ -99,13 +87,30 @@ public void delete() {
             }
     }
 
+    public void manageMinusOperator() {
+        if (firstOperand.isEmpty()) {
```

Comment:
We can use switch here and avoid the concatenation of ifs

Reply:
Can't be solve with a switch